ssl/statem/statem_srvr.c

```diff
@@ -1986,12 +2072,30 @@ static int tls_early_post_process_client_hello(SSL_CONNECTION *s)
         goto err;
     }
 
+#ifndef OPENSSL_NO_ECH
+    /*
+     * Unless ECH has worked or not been configured we won't call
+     * the session_secret_cb now because we'll need to calculate the
+     * server random later to include the ECH accept value.
+     * We can't do it now as we don't yet have the SH encoding.
+     */
+    if (((s->ext.ech.es != NULL && s->ext.ech.success == 1)
+         || s->ext.ech.es == NULL)
+        && (!s->hit
+            && s->version >= TLS1_VERSION
+            && !SSL_CONNECTION_IS_TLS13(s)
+            && !SSL_CONNECTION_IS_DTLS(s)
+            && s->ext.session_secret_cb != NULL)) {
+        const SSL_CIPHER *pref_cipher = NULL;
+#else
+
```

Comment:
I assume one option how to silence the style checker would be to do:
```suggestion
#elif 0
    /* Silence the style checker */
    }
#else
```

Reply:
Nice! Thanks, will try that.